modules/helm/Makefile.repo

```diff
@@ -0,0 +1,101 @@
+REPO_URL?=https://charts.cloudposse.com
+
+ifneq ($(TRAVIS_PULL_REQUEST_BRANCH),)
```

Comment:
Until we will have point where we can reuse this code I prefer to leave it in one place.